src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -3084,27 +3182,114 @@ def pv_capital_cost_constraint(c):
 
             @b.Constraint()
             def pv_capital_cost_constraint(c):
+                # no expenditure period, so cash flow occurs at t=0 (project year)
                 # PV_Capital_Cost = - CAPEX
 
                 return c.pv_capital_cost == -pyunits.convert(
-                    c.CAPEX
-                    * series_present_worth_factor(
+                    c.CAPEX,
+                    to_units=c.cost_units,
+                )
+
+        if b.config.debt_expression is None:
+
+            @b.Constraint()
+            def loan_debt_constraint(c):
+                # Debt  = %debt_charge_of_CAPEX * CAPEX
+
+                return c.loan_debt == pyunits.convert(
+                    pyunits.convert(
+                        c.debt_percentage_of_CAPEX, to_units=pyunits.dimensionless
+                    )
+                    * c.CAPEX,
+                    to_units=c.cost_units,
+                )
+
+        else:
+
+            b.loan_debt = Reference(b.config.debt_expression)
+
+        @b.Constraint()
+        def loan_annual_payment_constraint(c):
+            # Annual Loan Payment = Debt * A/P(%interest, 0, loan_length) = Debt / P/A(%interest, 0, loan_length)
```

Comment:
This conversion is from present value to future annual payments. However, I believe we would need to convert this back in order to get the present value of annual payments.